### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by gvisani** [gvisani/enzyme_promiscuity#1]
1. Add comments to all the code
2. Create similarity models. Must adapt it to allow for customized test set.
3. Test all models in this polished version
4. Save pretrained models. Maybe train new ones?
